compositor/sessionmanager/sessionmanager.cpp

```diff
@@ -201,15 +193,9 @@ bool SessionManager::canPowerOff()
     return m_powerManager->capabilities() & PowerManager::PowerOff;
 }
 
-bool SessionManager::canRestart()
-{
-    return m_powerManager->capabilities() & PowerManager::Restart;
-}
+bool SessionManager::canRestart() { return m_powerManager->capabilities() & PowerManager::Restart; }
```

Comment:
i'm concerned about this formatting, it's not very readable and not in the Qt style